### PR TITLE
JAMES-2993 Factorize documentation for returned taskIds

### PR DESCRIPTION
**Pull request by chibenwa**
While re-organizing the documentation to be resource orianted, and not RPC oriented, I was tired to have to handle duplicated TaskId response documentation...

### CODE REVIEW

File: src/site/markdown/server/manage-webadmin.md [hunk 1]
```diff
@@ -3330,6 +3214,37 @@ Response codes:
...
+
+ - Location header indicates the location of the resource associated with the scheduled task. Example:
+
+```
+Location: /tasks/3294a976-ce63-491e-bd52-1b6f465ed7a2
+```
+
+Response codes:
+
+ - 201: Success. Corresponding task id is returned.
```
Comment (trantienduchn) [inline src/site/markdown/server/manage-webadmin.md]: s/Success/Task genrated or Task created ?

Comment (chibenwa) [reply src/site/markdown/server/manage-webadmin.md]: I don't understand your proposal.

File: src/site/markdown/server/manage-webadmin.md [hunk 2]
```diff
@@ -3330,6 +3214,37 @@ Response codes:
...
+ - Other response codes might be returned depending on the endpoint
+
+The additional information of such tasks depends on the scheduled task and is documented in the endpoint documentation.
```
Comment (remk) [inline src/site/markdown/server/manage-webadmin.md]: The additional information returned depends on the scheduled task type and is documented in the endpoint documentation.
WDYT ?

File: src/site/markdown/server/manage-webadmin.md (hunk 1 above)
Comment (chibenwa) [reply src/site/markdown/server/manage-webadmin.md]: Ok I got your point. Will do in a separate commit.

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**